ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -44,31 +50,53 @@
  * the beginning until now, or any new messages since then), and writing to it.
  */
 
+// CHECKSTYLE_RULES.OFF: ClassDataAbstractionCoupling
```

Comment:
Do you still need to turn this rule off with Hojjat's work to split the CommandTopic out?

Reply:
Shouldn't be, but I'm merging first. Will coordinate with @hjafarpour to have to it removed.